overlord/snapstate/handlers.go

```diff
@@ -3163,7 +3165,7 @@ func (m *SnapManager) stopSnapServices(t *state.Task, _ *tomb.Tomb) error {
 	// no longer present.
 	snapst.LastActiveDisabledServices = append(
 		snapst.LastActiveDisabledServices,
-		disabledServices...,
+		disabledServices.SystemServices...,
```

Comment:
not for this PR, but we will need to be careful what we do here in terms of backward compat with state/changes coming from older snapd and when snapd is potentially reverted